Documentation/Administration/DirectoryStructure/SiteFolder.rst

```diff
@@ -0,0 +1,100 @@
+.. include:: /Includes.rst.txt
+
+..  _site-folder:
+
+===========
+Site folder
+===========
+
+The site folder (:file:`config/sites/my-site` in Composer-based installations,
+:file:`typo3conf/sites/my-site` in Classic installations) **must** contain the
+following file:
+
+..  typo3:file:: config.yaml
+    :name: site-config-yaml
+    :scope: site
+    :composerPath: /config/sites/my-site/
+    :classicPath: /typo3conf/sites/my-site/
+    :regex: /^.*(config|typo3conf)\/sites\/[\w-]+\/config\.yaml$/
```

Comment:
Not sure about that RegExp. The dash `-` is a special char for character classes. Needed for `A-Z` or `0-9`. Not tested and don't know, if I'm true, but I would expect:

```suggestion
    :regex: /^.*(config|typo3conf)\/sites\/[\w\-]+\/config\.yaml$/
```

Reply:
The other version is tested and working...

Reply:
Interesting: `but it does no harm if they are escaped`

Quote from PHP docs:

```
All non-alphanumeric characters other than \, -, ^ (at the start) and the terminating ] are non-special in character classes, but it does no harm if they are escaped.
```